docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/EndpointToOperationResponse.scala

```diff
@@ -104,4 +106,11 @@ private[openapi] class EndpointToOperationResponse(
       .defaultDecodeFailureOutput(input)
       .map(output => outputToResponses(output, ResponsesDefaultKey, None))
       .getOrElse(ListMap())
+
+  private def extractFixedContentType(outputs: List[EndpointOutput[_]]): Option[String] = {
+    outputs.flatMap(_.traverseOutputs {
+      case EndpointIO.FixedHeader(h, _, _) =>
+        if (h.name.equalsIgnoreCase("Content-Type")) Vector(Option(h.value)) else Vector(None)
```

Comment:
simpler: `Vecotr(h.value)` & `Vector.empty`. Then you just need to call `.headOption` on the result to get the first element (if any)